### DISCUSSION TITLE
How to make alternate account redeem codes on main account?

### DISCUSSION THREAD

**Discussion by Unknownisgod** [marioparaschiv/nitro-sniper#269] [Q&A]
How do I make it so that my alternate account redeems the captured codes on my main account? The description says it's a feature
Can someone give step by step explanation?

**Comment by MrD4rio** [comment]
It's a default feature, whatever token you have set as main into the config, all the codes will be redeemed on that one.

**Reply by Unknownisgod** [reply]
But what if I want to make it so that only my alt snipes code and them main one doesn't do anything like no participation in giveaways and no redeeming codes on main?
